samples/java-spring-choreography-saga-quickstart/README.md

```diff
@@ -0,0 +1,182 @@
+curl localhost:9000/api/users/001 \
+  --header "Content-Type: application/json" \
+  -XPOST \
+  --data '{ "name":"John Doe","country":"Belgium", "email":"doe@acme.com" }'
+```
+
+Check the logs of `UniqueEmailSubscriber` and `UserEventsSubscriber` to see how the saga is progressing.
+
+### check status for email doe@acme.com
+
+```shell
+curl localhost:9000/api/emails/doe@acme.com
+```
+The status of the email will be RESERVED or CONFIRMED, depending on whether the saga has been completed or not. 
+
+### create user identified by 002
+
+```shell
+curl localhost:9000/api/users/002 \
+  --header "Content-Type: application/json" \
+  -XPOST \
+  --data '{ "name":"Anne Doe","country":"Belgium", "email":"doe@acme.com" }'
+```
+A second user with the same email address will fail.
+
+### try to create an invalid user
+
+```shell
+curl localhost:9000/api/users/003 \
+  --header "Content-Type: application/json" \
+  -XPOST \
+  --data '{ "country":"Belgium", "email":"invalid@acme.com" }'
+```
+
+Note that the 'name' is not stored. This will result in the email address `invalid@acme.com` being reserved but not confirmed.
+
+Check the logs of `UniqueEmailSubscriber` to see how the saga is progressing.
+
+### check status for email invalid@acme.com
+
+```shell
+curl localhost:9000/api/emails/invalid@acme.com
+```
+
+The status of the email will be RESERVED or NOT_USED, depending on whether the timer to un-reserve it has fired or not.
+
+### Bonus: change an email address
+
+Change the email address of user 001 to `john.doe@acme.com`. Inspect the code to understand how it re-uses the existing saga.
+
+```shell
+curl localhost:9000/api/users/001/change-email \
+  --header "Content-Type: application/json" \
+  -XPUT \
+  --data '{ "newEmail": "john.doe@acme.com" }'
+```
+
+Check the logs of `UniqueEmailSubscriber` and `UserEventsSubscriber` to see how the saga is progressing.
+
+### check status for email doe@acme.com
+
+```shell
+curl localhost:9000/api/emails/doe@acme.com
+```
+
+The status of the email will be CONFIRMED or NOT_USED, depending on whether the saga has been completed or not.
+
+## Deploying
+
+To deploy your service, install the `kalix` CLI as documented in
+[Setting up a local development environment](https://docs.kalix.io/setting-up/)
+and configure a Docker Registry to upload your docker image to.
+
+You will need to update the `dockerImage` property in the `pom.xml` and refer to
+[Configuring registries](https://docs.kalix.io/projects/container-registries.html)
+for more information on how to make your docker image available to Kalix.
+
+Finally, you can use the [Kalix Console](https://console.kalix.io)
+to create a project and then deploy your service into the project either by using `mvn deploy kalix:deploy` which
+will conveniently package, publish your docker image, and deploy your service to Kalix, or by first packaging and
+publishing the docker image through `mvn deploy` and then deploying the image
+through the `kalix` CLI.
```

Comment:
- Do not provide two options for deploying service into project. Eliminate any unnecessary decisions the user has to make.
- Kalix Console cannot be used to create a project.

```suggestion
- Finally, use the `kalix` CLI to generate a project.
- Deploy your service into the project using `mvn deploy kalix:deploy`. This command conveniently packages, publishes your Docker image, and deploys your service to Kalix.
```